src/stylesheets/theme/styles.scss

```diff
@@ -51,8 +52,8 @@
 // @import 'packages/usa-nav-container';
 // @import 'packages/usa-nav';
 // @import 'packages/usa-navbar';
-// @import 'packages/usa-section';
 // @import 'packages/usa-search';
+// @import 'packages/usa-section';
 // @import 'packages/usa-sidenav';
```

Comment:
I think we're missing `packages/_usa-step-indicator.scss`

Reply:
I will be pleased to be done with this file...

Reply:
may as well add identifier as well

Reply:
Yeah, I wish we could glob it and bring only the components in.